composer.json

```diff
@@ -24,7 +24,8 @@
 	"require": {
 		"php": "^5.6.20 || ^7.0 || ^8.0",
 		"composer/installers": "^1.12.0",
-		"yoast/i18n-module": "^3.1.1"
+		"yoast/i18n-module": "^3.1.1",
+		"chillerlan/php-qrcode": "^2.0"
```

Comment:
We should probably vendor-prefix this to avoid potential conflicts if other plugins also use this library on an older version.

Reply:
Should probably downgrade it to 1.0.9 as well, that ensures PHP 5.6 compat without any additional effort from our end.